guides/common/modules/proc_installing-fapolicyd-on-server.adoc

```diff
@@ -23,9 +23,7 @@ endif::[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-ifndef::foreman-deb[]
-# dnf install fapolicyd
-endif::[]
+# {client-install-package-el8} fapolicyd
```

Comment:
I don't quite understand why you're using a `client-install-package*` attribute, when we're installing something on ProjectServer here. Shouldn't you use a `project-install-package` attribute in this case instead?

Reply:
In this special case, you install a package *before* installing Foreman. This is a problem for Satellite builds, because you overwrite the "project-install" attribute. This would mean that users try to install a package like "satellite-maintain install X" *before* Satellite is even installed. At least this is my understanding why "dnf install" was here in the first place. It's probably a good idea to add a small comment to the code block or even revert to "dnf install"?

Reply:
Oh, I see. I'm for reverting to a hard-coded value in this case.

Reply:
Reverted!